src/widgets/HistogramWidget.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect, useCallback } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { PropTypes } from 'prop-types';
-import { addFilter, removeFilter, selectSourceById } from '../redux/cartoSlice';
+import { addFilter, removeFilter, selectSourceById, selectPostsByUser } from '../redux/cartoSlice';
```

Comment:
```suggestion
import { addFilter, removeFilter, selectSourceById } from '../redux/cartoSlice';
```

Reply:
We should definitely add `lint-staged` to avoid this kind of things

Reply:
There is a ticket created